php73/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN ln -sfv ${PHP_HOME}/root/usr/bin/* /usr/bin/ && \
     ln -sfv /dev/stderr /var${PHP_HOME}/log/php-fpm/error.log
 
 # Install phpredis
-ENV PHPREDIS_VERSION 3.1.2
+ENV PHPREDIS_VERSION 5.3.1
```

Comment:
Are comfortable with compatibility with this level of jump? For the php 7.4 branch I had to move it to 3.1.5 but I only picked that as a "closest to the one that we have that works" type approach. If various testing has shown that this jump appears safe I have no problem merging.